browser/src/ConstraintTable/GnomadNonCodingConstraintTableVariant.tsx

```diff
@@ -15,11 +17,6 @@ const Table = styled(BaseTable)`
     td,
     th {
       padding-right: 10px;
-
-      /* Drop sparkline column */
-      &:nth-child(5) {
```

Comment:
@rileyhgrant @mattsolo1 we should talk sometime about how much responsive design matters to us. I could see, for instance, people using tablets in the lab fairly often.

Reply:
Oops, truthfully these styles were mimicking the gnomadConstraintTable. The genomic constraint table doesn't have a standalone sparkline column that should be dropped in smaller views.

I'd be interested in have a responsive design discussion, and refactoring. I agree with your suggestion for this piece of code, though I think (since I monkey see - monkey do'd) this feedback would be put into place in a different file. In this one, removing it was sufficient I believe.